databases.go

```diff
@@ -276,10 +279,15 @@ type evictionPolicyRoot struct {
 	EvictionPolicy string `json:"eviction_policy"`
 }
 
+type sqlModeRoot struct {
+	SQLMode string `json:"sql_mode"`
```

Comment:
It seems like the API returns a comma separated list as one string here. Seems like an array might be easier to use?

```
{
  "sql_mode": "ANSI,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION,NO_ZERO_DATE,NO_ZERO_IN_DATE,STRICT_ALL_TABLES"
}
```

Reply:
Yes, that's definitely a possibility. This setting is usually specified as a single string when working with MySQL, but there's no reason we can't present it differently or better. It sounds like @bentranter had a similar idea.